default_configure_options.bionic

```diff
@@ -0,0 +1,49 @@
+--enable-intl
```

Comment:
Doesn't this one also need all the version specific variations, at least for baseline, 7.4, master? (Which is also the reason why I still think that #29 is a more scalable solution to this problem.)

Reply:
sorry, i borrowed this from the xenial base line default for my testing.